packages/yarnpkg-core/sources/LightReport.ts

```diff
@@ -13,7 +13,7 @@ export type LightReportOptions = {
   suggestInstall?: boolean,
 };
 
-export class LightReport extends Report {
+export class LightReport extends ConfigurableReport {
```

Comment:
Btw, what if instead of extending `ConfigurableReport` we added some kind of mixin? For instance:

```ts
class MyReport extends Report {
  constructor({configuration}) {
    formatUtils.addLogFilterSupport(this, configuration);
  }
}
```

Since we can only inherit one class, I feel like perhaps this would be slightly more future-proof. Wdyt?

Reply:
This function would have to modify functions on the logger instance, and it would be impossible for the logger to explicitly call the actual logger function instead of the mixed-in function.

In the end it should be fine, but having e.g. `└ Completed` pass through the log filter seems wrong?